animation/animation.ino

```diff
@@ -0,0 +1,394 @@
+// Animation List
+// 1. Vibe Flash
+// 2. Vibe Rainbow
+// 3. Vibe Cursor Spin
+// 4. Truck Charge 
+// 5. Vibe Pulse
+// 6. Vibe Fireworks
+// 7. Vibe Sparkle
+///////////////////////////////////////
+
+
+///////////////////////////////////////
+// 1. Vibe Flash animation
+//// Animation outputs colour on all LEDS when button is pressed. Sets all LEDs to black when button is depressed
+///////////////////////////////////////
+void vibe_flash()
+{
+    if(is_button_pressed) {
+      for(int i = 0; i < NUM_LEDS; i++){
+		uint8_t bri = 255;
+		leds[i] = CHSV(global_hue, 200, bri);
+	}
+  }
+    else {
+      for(int i = 0; i < NUM_LEDS; i++){
+      leds[i] = CHSV(0, 0, 0);
+      } //If no button press, LEDs off
+    }
+    FastLED.show();
+  
+  //Color change if button is released
+  if(is_button_pressed == false & last_buttonpress == true) {
+    global_hue = random8();
+    Serial.println(global_hue);
+  }
+  last_buttonpress = is_button_pressed;
+}
+
+///////////////////////////////////////
+// 2. Vibe Rainbow animation
+//// Animation displays spinning rainbow through LEDS when button is pressed. Sets all LEDs to black when button is depressed
+///////////////////////////////////////
+void rainbow_vibe()
+{
+    if(is_button_pressed) {
+     fill_rainbow( leds, NUM_LEDS, global_hue, 1); // If button pressed, light up rainbow
+  }
+    else {
+      for(int i = 0; i < NUM_LEDS; i++){
+      leds[i] = CHSV(0, 0, 0);
+      } //If no button press, LEDs off
+    }
+    FastLED.show();
+  
+  //Hold colors if continuous press
+  if(is_button_pressed == true & last_buttonpress == true) {
+    global_hue = global_hue + 3;
+  }
+  last_buttonpress = is_button_pressed;
+}
+
+///////////////////////////////////////
+// 3. Vibe Cursor Spin animation
+//// Animation displays a cursor of leds that cycle through the strip when button pressed. Sets all LEDs to black when button is depressed
+///////////////////////////////////////
+void vibe_cursor_spin()
+{
+  // a colored dot moving around a cicle, with fading trails
+  if(is_button_pressed){
+  fadeToBlackBy( leds, NUM_LEDS, 3);
+  leds[pos] += CHSV( global_hue, 255, 255);
+  }
+  else {
+      for(int i = 0; i < NUM_LEDS; i++){
+      leds[i] = CHSV(0, 0, 0);
+      } //If no button press, LEDs off
+    }
+  FastLED.show();
+  
+if (pos >= 255){
+  pos = 0;
+  } //reset position of cursor back to 0 to spin it around the hoop
+
+  if(is_button_pressed == true & last_buttonpress == true){
+  pos++;
+  
+  }
+  //
+  if(is_button_pressed == false & last_buttonpress == true) {
+    global_hue = random8();
+  }
+  last_buttonpress = is_button_pressed; 
+}
+
+///////////////////////////////////////
+// 4. Trunk charge animation
+//// Upon press, "charge up" the LEDs on the tree by turning on LEDs in series until reaching full bar
+///////////////////////////////////////
+void trunk_charge(){
+  // Charge up the trunk with a green light
+  if(is_button_pressed){
+  leds[pos] = CHSV(96, 237, 255);
```

Comment:
Interesting Saturation value here(237) curious why that one? is this color matching a color palette someonewhere? If so leave a comment with where to find it so other contributors don't much with the color matching